README.md

```diff
@@ -15,10 +15,13 @@
     <a href="LICENSE">
         <img src="http://img.shields.io/badge/license-MIT-brightgreen.svg" alt="MIT License">
     </a>
-    <a href="https://github.com/vapor/queues-redis-driver/actions">
-        <img src="https://github.com/vapor/queues-redis-driver/workflows/test/badge.svg" alt="Continuous Integration">
+    <a href="https://github.com/vapor/queues-redis-driver/actions/workflows/test.yml">
```

Comment:
Though, why is here 2 swift 5.6 and swift 5.8 badges? what about the poor 5.7?!

Reply:
I (obviously) like DiscordBM's approach which is dividing the versions by a " / " (I don't recall where i copied that from 😅):
```html
<a href="https://swift.org">
        <img src="https://img.shields.io/badge/swift-5.8%20/%205.7-brightgreen.svg" alt="Latest/Minimum Swift Version">
    </a>
```

Reply:
Hm... I definitely get the appeal of the "divider" approach. At the same time, I'm leaning towards changing my previous intent of showing both minimum and latest in favor of showing only the minimum, on the theory that the badge arguably suggests a claim that the package "has been updated for" the version on the badge, and that's certainly not the case for 5.8 in any package (with the exception of the `Exports.swift` bit), or even 5.7 in many of them. Thoughts?

Reply:
Yeah that seems reasonable :)
Generally the subject is subjective (😅) so I was just mentioning that there are alternative ways. But it still makes sense why they are there, like you said.